app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post, counter_cache: true
+  def update_counter
+    post.update(comments_count: post.comments.size)
+  end
+end
```

Comment:
- [ ] Again here It would be nice to use one of the Rails active callbacks after_save to call the #update_counter method as suggested in the comment above. Kindly do so.